pydisdrometer/plot/plot.py

```diff
@@ -32,22 +34,39 @@ def plot_dsd(dsd, range=None, log_scale=True, tighten=True):
     fig_handle: Figure Handle
 
     '''
+    ax =  parse_ax(ax)
+    fig = parse_ax(fig)
```

Comment:
Yep, I did, changed it. Why would this create two plots? The plt.gcf() get the figure reference.